kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -147,6 +147,12 @@ usageRightsEditor.controller(
 
     ctrl.cancel = () => ctrl.onCancel();
 
+    $scope.$watch(() => ctrl.showRestrictions, onValChange(restrictions => {
```

Comment:
Could we do this on Save rather than here? That way you can toggle the restriction and not loose the text; you'd only lose it after you save?